bcipy/helpers/load.py

```diff
@@ -162,6 +162,19 @@ def load_experimental_data() -> str:
     return filename
 
 
+def load_eye_tracking_data(gaze_data: RawData):
```

Comment:
`def load_eye_tracking_data(gaze_data: RawData) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:`

Reply:
This function was redundant, so I ended up deleting it.